sopel/cli/plugins.py

```diff
@@ -349,7 +374,14 @@ def display_unknown_plugins(unknown_plugins):
 
 
 def handle_disable(options):
-    """Disable Sopel plugins"""
+    """Disable Sopel plugins
+
+    :param options: parsed arguments
+    :type options: ``argparse.Namespace``
+    :return: Return 0 if everything went fine.
+             Return 1 for coretasks (cannot be disabled)
+             or if the plugin doesn't exist.
```

Comment:
```suggestion
    :return: 0 if everything went fine;
             1 if the plugin doesn't exist,
             or if attempting to disable coretasks (required)
```

I haven't test-rendered this to see what it looks like though.